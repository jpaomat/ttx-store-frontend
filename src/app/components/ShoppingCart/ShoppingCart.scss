.shopping-cart__counter {
    position: absolute;
    width: 22px;
    height: 22px;
    border-radius: 100%;
    top: 10px;
    right: 10px;
    background: #7028FC;
    display: flex;
    align-items: center;
    justify-content: center;
    font-weight: 700;
    font-family: Lato,sans-serif;
    border: 1px solid #ffffff;
    color: #ffffff;
    font-size: 12px;
    padding-right: 1px;
}

@media (min-width: 1024px) {
    .shopping-cart__counter {
        // top: -3px;
        // right: 5%;
    }
}