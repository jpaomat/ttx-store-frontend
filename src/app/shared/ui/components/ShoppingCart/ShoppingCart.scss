.shopping-cart__counter {
    position: absolute;
    width: 22px;
    height: 22px;
    border-radius: 100%;
    top: 7px;
    right: 10px;
    background: rgb(47, 79, 79);
    display: flex;
    align-items: center;
    justify-content: center;
    font-weight: 700;
    font-family: Lato,sans-serif;
    border: 1px solid #ffffff;
    color: #ffffff;
    font-size: 12px;
    padding-right: 1px;
}

@media (min-width: 1024px) {
    .shopping-cart__counter {
        // top: -3px;
        // right: 5%;
    }
}