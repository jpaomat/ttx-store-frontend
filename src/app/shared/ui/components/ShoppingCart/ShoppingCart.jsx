import PropTypes from 'prop-types';
import { BsCart4 } from 'react-icons/bs';
import './ShoppingCart.scss';

export const ShoppingCart = ({ items }) => {

	return (
		<>
			<BsCart4 size={30} color='rgb(47, 79, 79)' />
			<span className='shopping-cart__counter'>{items}</span>
		</>
	);
};

ShoppingCart.defaultProps = {};

ShoppingCart.propTypes = {
	items: PropTypes.number.isRequired,
};
