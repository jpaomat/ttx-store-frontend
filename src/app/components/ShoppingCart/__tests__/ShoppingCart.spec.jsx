import { ShoppingCart } from '../ShoppingCart';
const Renderer = require('react-test-renderer/shallow');

const renderer = new Renderer();

describe('Techtronix component', () => {
	it('should render component', () => {
		const component = renderer.render(<ShoppingCart items={5} />);
		expect(component).toMatchSnapshot();
	});
});