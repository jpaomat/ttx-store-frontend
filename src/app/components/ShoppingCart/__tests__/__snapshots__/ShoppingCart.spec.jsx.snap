// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Techtronix component should render component 1`] = `
<React.Fragment>
  <BsCart4
    color="rgb(47, 79, 79)"
    size={30}
  />
  <span
    className="shopping-cart__counter"
  >
    5
  </span>
</React.Fragment>
`;
